src/Entities/Simulation.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Paddle\SDK\Entities;
+
+use Paddle\SDK\Entities\Shared\JSONObject;
+use Paddle\SDK\Entities\Simulation\SimulationScenarioType;
+use Paddle\SDK\Entities\Simulation\SimulationSingleEventType;
+use Paddle\SDK\Entities\Simulation\SimulationStatus;
+
+class Simulation implements Entity
+{
+    private function __construct(
+        public string $id,
+        public SimulationStatus $status,
+        public string $notificationSettingId,
+        public string $name,
+        public SimulationSingleEventType|SimulationScenarioType $type,
+        public JSONObject|null $payload,
```

Comment:
Should this be type `NotificationEntity`?

Reply:
^ ignore this, I miss remembered the interface usage for `Notifications\Entity` and actually this I think would be fine to use